src/spaces/okhsl.js

```diff
@@ -458,6 +458,11 @@ function oklabToOkhsl (lab, lmsToRgb, okCoeff) {
 		}
 	}
 
+	if (Math.abs(s) < 1e-4 || l === 0.0 || Math.abs(1 - l) < 1e-7) {
```

Comment:
Where do these constants come from?

Reply:
It's an approximation. Considering that Okhsl and Okhsv is mainly 32 bit accurate, I only require the lightness to be close to that.

Generally, everywhere else in this library, close to zero for chroma is not required to be super close to actual zero. I think originally I ran some forward and backwards conversion between various color spaces and settled on 1e-4 for "sufficiently close". I don't recall the exact reason I selected that value, but that is what I did.

Generally, even the CSS spec does not specify what "sufficiently close to zero" is:

> It is intentionally unspecified exactly what "sufficiently close" means for this purpose.

Now that I look into Color.js, OkLCh uses the arbitrary value of 2e-4 to determine achromatic status, so if you prefer that, I can use that: https://github.com/color-js/color.js/blob/main/src/spaces/oklch.js#L31C13-L31C19. The difference is insignificant, but I can see it making sense for the sake of consistency.

Reply:
I will also add that `l === 0.0` is a more reliable target than `l === 1.0`. This is generally true for _most_ color space algorithms. It is often observable that getting a precise 1 for lightness or 100 for lightness is algorithmically less likely due to floating point math than targeting a lightness of zero during conversion.

Reply:
Can we abstract them away to some kind of ε constant?

Reply:
Sure.